core/src/main/java/org/elasticsearch/search/suggest/phrase/PhraseSuggestionBuilder.java

```diff
@@ -610,7 +614,6 @@ public SuggestionContext build(QueryShardContext context) throws IOException {
         suggestionContext.setRealWordErrorLikelihood(this.realWordErrorLikelihood);
         suggestionContext.setConfidence(this.confidence);
         suggestionContext.setMaxErrors(this.maxErrors);
-        suggestionContext.setSeparator(BytesRefs.toBytesRef(this.separator));
```

Comment:
Duplication of https://github.com/elastic/elasticsearch/pull/25571/files#diff-5520403586f385d80a2630ef8a9f7404R614